app/templates/_profile_prod.gradle

```diff
@@ -30,6 +30,8 @@ task setProdProperties(dependsOn: bootRun) << {
     }
 }
 
+setProdProperties.dependsOn 'gulp_ngconstant:prod'
```

Comment:
I think for prod it should depend on gulp build to ensure that all prod tasks are done